module-4/backend/week_two.md

```diff
@@ -5,17 +5,37 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is Webpack and why is it useful?
+  - Webpack does the job of Rails' Asset Pipeline, prepping your code for better web transfer by compiling files into one file, minifying, uglifying, etc.  
 2. When do you want to use event delegation?
+  - When you want to check something dynamically created, you need to delegate event listening to a parent element, as listeners are bound only on elements that exist when the document is ready. 
+  
 3. What's one difference between ES5 and ES6?
+- Syntax is significantly different, the context of `this` also changes in some instances. 
```

Comment:
Certainly.  Arrow syntax in ES6 is the first thing that comes to mind. ES6 also has a more object-oriented syntax when it comes to constructors, using a `class` keyword and defining functions without the `.prototype`
ES6 syntax also allows default parameters, which is awesome.

Oh, best of all about ES6: Semicolons are optional!